README.md

```diff
@@ -41,44 +67,47 @@ Include a diagram describing the application
 - This model uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
     - We recommend using [SUGGESTED REGION]
 - ...
- 
-### Installation
- 
+
+#### Installation
+
 (ideally very short)
  
 - npm install [package name]
 - mvn install
 - ...
- 
-### Quickstart
+
+#### Quickstart
+
 (Add steps to get up and running quickly)
  
 1. Clone the repository and intialize the project: `azd init [name-of-repo]`
 2. ...
 3. Provision and deploy the project to Azure: `azd up`
 4. Set up CI/CD with `azd pipeline config`
```

Comment:
This section has azd pipeline config, but not the earlier sections about codespaces/VS Code. Either we need to repeat it in both places, or all sections jump to a separate section about deploying. (That's the tricky part of having the Codespaces/Dev Containers instructions, that jump).